node/src/chain_spec.rs

```diff
@@ -124,101 +89,93 @@ pub fn development_config() -> Result<ChainSpec, String> {
 					get_account_id_from_seed::<sr25519::Public>("Eve//stash"),
 					get_account_id_from_seed::<sr25519::Public>("Ferdie//stash"),
 				],
-				true,
+				id,
 			)
 		},
-		// Bootnodes
 		vec![],
-		// Telemetry
 		None,
-		// Protocol ID
 		None,
-		// Properties
 		None,
-		// Extensions
-		None,
-	))
+		Extensions {
+			relay_chain: "rococo-local".into(), // You MUST set this to the correct network!
+			para_id: id.into(),
+		},
+	)
 }
 
-pub fn testnet_config() -> Result<ChainSpec, String> {
-	let wasm_binary = WASM_BINARY.ok_or_else(|| "Development wasm not available".to_string())?;
-	let mut properties = sc_service::Properties::new();
-	properties.insert("tokenSymbol".to_string(), "CAN".into());
-	properties.insert("tokenDecimals".to_string(), 12.into());
-	properties.insert("ss58Format".to_string(), 42.into());
+pub fn local_testnet_config(id: ParaId) -> ChainSpec {
+	// Give your base currency a unit name and decimal places
+	let mut properties = sc_chain_spec::Properties::new();
+	properties.insert("tokenSymbol".into(), "CAN".into());
+	properties.insert("tokenDecimals".into(), 12.into());
 
-	Ok(ChainSpec::from_genesis(
+	ChainSpec::from_genesis(
+		// Name
 		"Canvas Testnet",
-		"canvas_testnet2",
-		ChainType::Live,
+		// ID
+		"canvas_testnet",
+		ChainType::Local,
 		move || {
+			// TODO: Right now these are being left as default values, but in the old `canvas-node`
+			// we had some non-default authorities, as well as accounts for the faucent and
+			// `ink-waterfall`. We'll need to add those back in
 			testnet_genesis(
-				wasm_binary,
-				testnet_authorities(),
-				testnet_root(),
+				get_account_id_from_seed::<sr25519::Public>("Alice"),
+				vec![get_from_seed::<AuraId>("Alice"), get_from_seed::<AuraId>("Bob")],
 				vec![
-					testnet_root(),
-					// AccountId of the Canvas Testnet faucet
-					hex!("18c64aa111a8a0e6e4eed41d6d906c7614d745e48be3cfc13b6128e1d51f4405").into(),
-					// AccountId of an account which `ink-waterfall` uses for automated testing
-					hex!("0e47e2344d523c3cc5c34394b0d58b9a4200e813a038e6c5a6163cc07d70b069").into(),
+					get_account_id_from_seed::<sr25519::Public>("Alice"),
+					get_account_id_from_seed::<sr25519::Public>("Bob"),
+					get_account_id_from_seed::<sr25519::Public>("Charlie"),
+					get_account_id_from_seed::<sr25519::Public>("Dave"),
+					get_account_id_from_seed::<sr25519::Public>("Eve"),
+					get_account_id_from_seed::<sr25519::Public>("Ferdie"),
+					get_account_id_from_seed::<sr25519::Public>("Alice//stash"),
+					get_account_id_from_seed::<sr25519::Public>("Bob//stash"),
+					get_account_id_from_seed::<sr25519::Public>("Charlie//stash"),
+					get_account_id_from_seed::<sr25519::Public>("Dave//stash"),
+					get_account_id_from_seed::<sr25519::Public>("Eve//stash"),
+					get_account_id_from_seed::<sr25519::Public>("Ferdie//stash"),
```

Comment:
Just double checking: This is only for a local testnet that developers are spawning and not what is used on rococo? The rococo parachchain spec is in `canvas-node/res/testnet-1.json`, right? No changes there?

Reply:
So the content of this function `local_testnet_config` is what was used for the testnet, the json is just the output of `cargo run -- build-spec --raw > res/testnet-1.json`, making it easier for DevOps.  We should rename this function to something different to make it clearer that this is the Rococo configuration.

The faucet and `ink-waterfall` are the most important ones, besides that it probably doesn't make sense to have other accounts for Rococo?

Reply:
> We should rename this function to something different to make it clearer that this is the Rococo configuration.

Yes. We should not have those accounts on rococo. They could be used by anyone.

Reply:
And we should remove `res/testnet-1.json` for now.

Reply:
As I understand it, if we run with a custom spec we won't be running as a `local` network anymore, so these values won't apply

Reply:
For posterity: Well known accounts will be removed later: https://github.com/paritytech/canvas-node/issues/76